plugins/ccgx/fu-ccgx-hpi-device.c

```diff
@@ -1380,8 +1378,14 @@ fu_ccgx_hpi_device_setup (FuDevice *device, GError **error)
 	}
 
 	/* set name to be more descriptive */
-	name = g_strdup_printf ("USB-I2C Bridge (%s)", fu_ccgx_hpi_device_get_name (self));
+	if (self->device_name) 
+		name = g_strdup_printf ("%s (%s)", self->device_name, fu_ccgx_hpi_device_get_fw_name (self));
+	else
+		name = g_strdup_printf ("USB-I2C Bridge (%s)", fu_ccgx_hpi_device_get_fw_name (self));
```

Comment:
I think if you can use the `Name` key in quirks, you can instead do `fu_device_get_name` here instead of `self->device_name`.  The set call will just update it to the right thing you want.

Reply:
If you end up removing the FW name from the title key, then this code can basically be:
```
if (fu_device_get_name (FU_DEVICE (self)) == NULL)
        fu_device_set_name (FU_DEVICE (self), "USB-I2C Bridge")
```

Reply:
Fixed it at [https://github.com/fwupd/fwupd/pull/2013/commits/d40f67f4c8af3ebfc2342f155d367eb7b6b73e15](url)